utils/docker/Dockerfile.build-gcc-plugins-deps

```diff
@@ -0,0 +1,18 @@
+ARG BASE=wasmedge/wasmedge:ubuntu-build-gcc
+FROM ${BASE}
+
+RUN apt update && apt install -y \
+  wget \
+  unzip
+
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN wget -O opencv.zip https://github.com/opencv/opencv/archive/refs/tags/4.8.0.zip
```

Comment:
Since the download and build steps are the same, should we put them into a standalone file?
Maybe use `ADD build_opencv.sh` and execute it.